CDTDatastore/HTTP/CDTURLSession.m

```diff
@@ -33,6 +34,7 @@ @implementation CDTURLSession
 
 - (instancetype)init
 {
+    self.asyncTaskMonitor = dispatch_semaphore_create(4);
```

Comment:
@brynh thanks, since fixed and squashed so the original commit is no longer there.